handlers/events.go

```diff
@@ -75,6 +80,39 @@ func (d *EventsHandlersCollection) Events() httprouter.Handle {
 	}
 }
 
+// ProxyEvents is a handler of Catalyst API called by Studio API.
+// It proxies the requests to Catalyst.
+func (d *EventsHandlersCollection) ProxyEvents() httprouter.Handle {
+	// Proxy the request to d.eventsEndpoint
+	return func(w http.ResponseWriter, req *http.Request, _ httprouter.Params) {
+		// Read the request body
+		body, err := io.ReadAll(req.Body)
```

Comment:
Could we not just forward the body into the new request rather than reading it into memory?

Reply:
Right. Updated in f4949ad7bb2ead7d6512b23fd4d4fd644efdafa2